fastlane/Fastfile

```diff
@@ -95,6 +95,19 @@ platform :android do
 
     take_android_emulator_screenshots(devices: screenshot_devices, screenshot_options: screenshot_options)
   end
+
+  #####################################################################################
+  desc "Rebuild screenshot devices"
+  lane :rebuild_screenshot_devices do |options|
+    sh("helpers/create-device.sh fastlane_screenshots_phone 17")
```

Comment:
are we sure we want to use hardcoded `device id`s here? looking into `helpers/create-device.sh` I can see we're passing the `17`, `33`, and `34` down for calling into `avdmanager create avd` with parameter `--device "${device_id}"`.

Here's what the help command throws for that option:
```
-d --device  : The optional device definition to use. Can be a device index
                 or id.
```
And, when listing devices, in my case I see this:

```
---------
id: 17 or "wear_square_240"
    Name: Android Wear Square
    OEM : Google
    Tag : android-wear
```

and these for 33 and 34 respectively:

```
---------
id: 33 or "2.7in QVGA"
    Name: 2.7" QVGA
    OEM : Generic
---------
id: 34 or "2.7in QVGA slider"
    Name: 2.7" QVGA slider
    OEM : Generic
```
which are not seven and ten inch as the device name we're creating in this script suggests.

Just tried it manually to double check, and in effect after running this command:
`$ANDROID_HOME/tools/bin/avdmanager --silent create avd -n "test_mario" -k "system-images;android-26;google_apis;x86" --device "17" --force`


I end up having this in my list:

```
---------
    Name: test_mario
  Device: wear_square_240 (Google)
    Path: /Users/mariozorz/.android/avd/test_mario.avd
  Target: Google APIs (Google Inc.)
          Based on: Android 8.0 (Oreo) Tag/ABI: google_apis/x86

```
It seems the `device id` may vary from computer to computer? Or should something else be configured as a pre-condition?